src/featureforest/models/DinoV2/model.py

```diff
@@ -0,0 +1,25 @@
+from typing import Tuple
+
+import torch
+
+# from featureforest.utils.downloader import download_model
+from .adapter import DinoV2Adapter
+
+
+def get_model(
+    img_height: float, img_width: float, *args, **kwargs
+) -> Tuple[DinoV2Adapter, torch.device]:
```

Comment:
Would it make sense to cache it the first time it is called (e.g. using `pooch` without a path so that it goes into the hidden folder in the user home)?

Reply:
The `torch.hub.load` also has its own cache. So, it won't download it again.  
Although I got the _DinoV2_ model URL (in comment line 17), having it downloaded with pooch, we also need the model source code which means we need to add another dependency.  
The downside of using torch hub is that I couldn't find a way to have progress of downloading the model via `torch.hub.load` (I haven't spent much time there).